code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -268,10 +268,14 @@
 			to_chat(M, "<span class='warning'>You need to unbuckle [src] first to do that!</span>")
 			return
 		M.visible_message("<span class='notice'>[M] shakes [src] trying to get [p_them()] up!</span>", \
-						"<span class='notice'>You shake [src] trying to get [p_them()] up!</span>")
+						null, "<span class='hear'>You hear the rustling of clothes.</span>", DEFAULT_MESSAGE_RANGE, list(M, src))
+		to_chat(M, "<span class='notice'>You shake [src] trying to pick [p_them()] up!</span>")

```

Comment:
why did you add these to chats? visible message has a self message, a receiver message and a deaf message, it should cover all cases.

Reply:
visible message covers anyone in sight, self, and blind people, so I figured using to_chat retains the sight based portion of viewing while the to_chats cover messages directly to people (I think some other attack_hand procs do something similar for blind feedback)